acceptance/repository-cloudant/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@loopback/test-repository-cloudant",
+  "version": "0.4.8",
+  "description": "",
+  "private": true,
+  "engines": {
+    "node": ">=8.9"
+  },
+  "scripts": {
+    "build": "lb-tsc",
+    "clean": "lb-clean loopback-test-repository-cloudant*.tgz dist tsconfig.build.tsbuildinfo package",
+    "pretest": "npm run build",
+    "test": "npm run mocha",
+    "mocha": "lb-mocha \"dist/__tests__/**/*.js\"",
+    "verify": "npm pack && tar xf loopback-test-repository-cloudant*.tgz && tree package && npm run clean",
+    "docker:setup": "node ./docker.setup.js"
+  },
+  "author": "IBM Corp.",
+  "copyright.owner": "IBM Corp.",
+  "license": "MIT",
+  "devDependencies": {
+    "@loopback/build": "^1.7.1",
+    "@loopback/eslint-config": "^4.1.2",
+    "@loopback/repository": "^1.15.2",
+    "@loopback/repository-tests": "^0.6.1",
+    "@types/node": "^10.17.5",
+    "loopback-connector-cloudant": "2.3.2",
+    "typescript": "~3.7.2",
+    "async": "^3.1.0",
+    "debug": "^4.1.1",
+    "lodash": "^4.17.11",
+    "dockerode": "^2.4.3"
```

Comment:
`ms` is missing from this list - it is used directly by `docker.setup.js`

Reply:
yeah I forgot.. but why is it still working? 0.0

Reply:
`ms` is a pretty common dependency. If any of the other dependencies depend on it then they would have pulled it in and due to dependency tree flattening, it would be installed as a top-level dependency.